src/library_webgpu.js

```diff
@@ -662,14 +680,28 @@ var LibraryWebGPU = {
       'sint32x2',
       'sint32x3',
       'sint32x4',
+      'unorm10-10-10-2',
     ],
     VertexStepMode: [
+      undefined,
+      'vertex-buffer-not-used',
       'vertex',
       'instance',
-      undefined,
     ],
   },
 
+  // Non-method functions
+
+  wgpuCreateInstance: (descriptor) => 1,
+
+  wgpuGetInstanceFeatures: () => {
+    abort('TODO: wgpuGetInstanceFeatures unimplemented');
+  },
+
+  wgpuGetProcAddress: (device, procName) => {
+    abort('TODO(#11526): wgpuGetProcAddress unimplemented');
+  },
```

Comment:
Reverting this change for now because
```
WARNING - [JSC_TYPE_MISMATCH] actual parameter 1 of BigInt does 
not match formal parameter
found   : undefined
required: (bigint|number|string)
  28102|     return BigInt(abort('TODO(#11526): wgpuGetProcAddress unimplemented'));
```
(They're expected to return values and abort doesn't)